sites/platform/static/files/fetch/examples/relationships/vault-kms

```diff
@@ -1,20 +1,24 @@
 {
     "username": "",
-    "scheme": "http",
-    "service": "vault-kms",
     "fragment": "",
     "ip": "169.254.196.95",
-    "hostname": "ckmpv2fz7jtdmpkmrun7yfgut4.vault-kms.service._.eu-3.platformsh.site",
-    "port": 8200,
     "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "vault-kms.internal",
-    "rel": "sign",
     "path": "\/",
     "query": {
         "is_master": true
     },
-    "password": "b.AAAAAQKY7LuvSZLHlkVOunFGip34HDGbXdOwcSzUK3Mkmdpp5TbGkw2KRaBGyuQRfvaI8h3jXG5iBYP3HcZMpk01uz72C3YTqdszglYXZlOtC0e3QU__T58dVEreodrVWGATYA4IxvIpSGaXvArC1hmA-c0CORPf-dsQQeswfO4sZayuC2WpLmUNXC7G9TKsOKuNxlS3l1byI5CQzuO7OEpD-rCjHvnewmFNDfmWGQ",
+    "password": "b.AAAAAQIm98Ptsj51HNBsgo4QUSJPFPdhAHCIML0ayQZOZsfRKLsXgh1gw4Tc5RbLjpJPKHdMtOObh-G1eNQwrgCei_wHPLciC7S-fxHRzv3t1ZqrxbnKIueuQPwE3-ur-wSrEmz3u08RfqKGAi0deP-LDwEt06j1qqJVwmHyMXTZYp5zOOB8B2nF129bbvX_Aom3Tqt_wI0x1uXW1yU4rdDdFc4SAuIcPfb_6rG08g",
+    "port": 8200,
+    "host_mapped": false,
+    "service": "vault-kms",
+    "hostname": "ckmpv2fz7jtdmpkmrun7yfgut4.vault-kms.service._.eu-3.platformsh.site",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.vault-kms.service._.eu-1.platformsh.site",